### PR TITLE
Moved Investigation schemas from CTIM to CTIA

### PR DESCRIPTION
**Pull request by gbuisson**
Hi, I worked on your PR branch to help you move out the Investigation schemas from CTIM so they are now defined in CTIA only.

- Moved the entity schema definition out of CTIM to CTIA
- Split the entity definition ns into sub namespaces
- Moved the Investigation examples

### CODE REVIEW

File: src/ctia/entity/investigation/examples.clj [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   :investigated_observables []
+   :targets []
+   :schema_version c/ctim-schema-version
+   :type "investigation"})
+
+(def new-investigation-maximal
+  investigation-maximal)
+
+(def new-investigation-minimal
+  {:source "a source"})
```
Comment (msprunck) [inline src/ctia/entity/investigation/examples.clj]: I would move these fixtures to the test folder

**Comment by noprompt** [comment]
@gbuisson Thanks for your help on this. After this is merged I can submit the migration patch.

**Comment by noprompt** [comment]
Are we ready to merge this or are we waiting for something?

**Comment by gbuisson** [comment]
hahaha I was wondering why you didn't merge it, sorry

**Comment by noprompt** [comment]
@gbuisson Ha! Sorry about the confusion, I didn't know I could merge it. 😄 Thank you!

**Pull request merged**